pkg/tsdb/prometheus/resource/resource.go

```diff
@@ -29,10 +29,12 @@ func New(
 		return nil, err
 	}
 	httpMethod, _ := maputil.GetStringOptional(jsonData, "httpMethod")
+	cacheLevel, _ := maputil.GetStringOptional(jsonData, "cacheLevel")
 
 	return &Resource{
 		log:        plog,
 		promClient: client.NewClient(httpClient, httpMethod, settings.URL),
+		cacheLevel: cacheLevel,
```

Comment:
Why do we need `cacheLevel` here? It seems we don't use it.

Reply:
Sorry about that, this was my initial implementation and I guess I forgot to remove it all